controllers/mover/restic/builder.go

```diff
@@ -196,6 +196,7 @@ func (rb *Builder) FromDestination(client client.Client, logger logr.Logger,
 		privileged:            privileged,
 		restoreAsOf:           destination.Spec.Restic.RestoreAsOf,
 		previous:              destination.Spec.Restic.Previous,
+		deleteFilesOnRestore:  destination.Spec.Restic.Delete,
```

Comment:
Not a fan of this name as it can easily be read as "we're going to delete all files in the PVC before we start the restore." `allowDeleteOnRestore`?

Reply:
I was struggling to come up with a good name for this...

I've been waffling back and forth about this api name and the name I use in the builder (ideally I'd like to use the same name in both places, but think I ended up with a different name to indicate this was a restore only parameter).  

I do think I like `enableFileDeletion` for the API -  So what do you think about `enableFileDeletionOnRestore` for the builder.go? @JohnStrunk